### ISSUE TITLE
Return type of generated docstrings is Any when it should not be

### ISSUE THREAD

**Issue by InvincibleRMC** [microsoft/pylance-release#6745]
## Environment data

-   Pylance version: v2024.12.1
-   OS and version: Ubuntu 24.04 LTS
-   Python version (& distribution if applicable, e.g. Anaconda): Python3.12

## Code Snippet

```python
def foo(a: int) -> float:
    """Docstring for foo
    
    :param a: Description
    :type a: int
    :return: Description
    :rtype: Any"""
    return 3.0
```

## Repro Steps

1. Create a function for ej..

```python
def foo(a: int) -> float:
    return 3.0
```

2. Use New generated docstrings

## Expected behavior

```python
def foo(a: int) -> float:
    """Docstring for foo
    
    :param a: Description
    :type a: int
    :return: Description
    :rtype: float"""
    return 3.0
```

## Actual behavior

```python
def foo(a: int) -> float:
    """Docstring for foo
    
    :param a: Description
    :type a: int
    :return: Description
    :rtype: Any"""
    return 3.0
```

**Comment by KacieKK** [comment]
Thanks for reporting, this is a known bug I am currently working on a fix.

**Comment by rchiodo** [comment]
This issue has been fixed in prerelease version 2024.12.100, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202412100-16-December-2024-PreRelease)
